Description: disable GBASIS_SWAP as it gives issues on some platforms
Author: Julien Puydt
Forwarded: better fix in discussion with upstream

--- giac.orig/src/cocoa.cc
+++ giac/src/cocoa.cc
@@ -441,9 +441,9 @@
   
 #if !defined CAS38_DISABLED && !defined FXCG
   //#define GBASIS_SELECT_TOTAL_DEGREE
-#if GROEBNER_VARS!=15 // double revlex ordering is not compatible with indices swapping
-#define GBASIS_SWAP 
-#endif
+//#if GROEBNER_VARS!=15 // double revlex ordering is not compatible with indices swapping
+//#define GBASIS_SWAP 
+//#endif
   // minimal numbers of pair to reduce simultaneously with f4buchberger
 #ifdef __APPLE__
   #define GBASISF4_BUCHBERGER 0 // temp. disabled on mac
@@ -14832,7 +14832,7 @@
     }
   }
 
-  #define GBASIS_SWAP 
+//  #define GBASIS_SWAP 
 #ifndef NO_STDEXCEPT
   #define GIAC_TDEG_T14
 #endif
